### PR TITLE
Add documentation for backfill ticket response

### PR DESCRIPTION
**Pull request by highlyunavailable**
Documentation for https://github.com/googleforgames/open-match/pull/1382

### CODE REVIEW

File: site/content/en/docs/Guides/Backfill/_index.md [hunk 1]
```diff
@@ -19,34 +19,32 @@ Some scenarios that could happen when playing multiplayer games include:
...
+In order to use Backfill you need to customize your MMF and your Game Servers. Backfill IDs should be propagated to each Game Server during Game Server allocation to be used in `AcknowledgeBackfill` requests to define Backfill which is tied to the Game Server.
+
 If Backfill is not being used, no changes are required since the Backfill API doesn't change 
 the behavior of the current Open Match API state.
 
 ### Acknowledging Backfills
 
-AcknowledgeBackfill is a request, that acts as a periodic ping, made to Open Match Frontend to set the assignment on the tickets associated with this backfill.
-GameServer receives current Backfill status as a response.
+`AcknowledgeBackfill` is a request that acts as a periodic heartbeat and is made to Open Match Frontend to set the assignment on the tickets associated with this backfill. The Game Server receives the current Backfill status as well as a list of new tickets that were just acknowledged by the request as a response. The list is not idempotent, but can be appended to prior lists received from `AcknowledgeBackfill`.
```
Comment (syntxerror) [inline site/content/en/docs/Guides/Backfill/_index.md:42]: Can we have a different word for heartbeat for this context?

Comment (highlyunavailable) [reply site/content/en/docs/Guides/Backfill/_index.md:42]: Sure, but I'd be interested in the reason!

Comment (highlyunavailable) [reply site/content/en/docs/Guides/Backfill/_index.md]: Wording changed, though.

Comment (syntxerror) [reply site/content/en/docs/Guides/Backfill/_index.md]: I've followed up offline in Slack

**Pull request merged**